spec/mailers/dependents_application_failure_mailer_spec.rb

```diff
@@ -3,28 +3,17 @@
 require 'rails_helper'
 
 RSpec.describe DependentsApplicationFailureMailer, type: [:mailer] do
-  let(:user_object) { FactoryBot.create(:evss_user, :loa3) }
-  let(:user_hash) do
-    {
-      'participant_id' => user_object.participant_id,
-      'ssn' => user_object.ssn,
-      'first_name' => user_object.first_name,
-      'last_name' => user_object.last_name,
-      'email' => user_object.email,
-      'external_key' => user_object.common_name || user_object.email,
-      'icn' => user_object.icn
-    }
-  end
+  let(:user) { FactoryBot.create(:evss_user, :loa3) }
 
   describe '#build' do
     it 'includes all info' do
-      mailer = described_class.build(user_hash).deliver_now
+      mailer = described_class.build(user).deliver_now
 
       expect(mailer.subject).to eq("We can't process your dependents application")
-      expect(mailer.to).to eq([user_object.email])
+      expect(mailer.to).to eq([user.email])
       expect(mailer.body.raw_source).to include(
-        "Hello #{user_object.first_name} #{user_object.last_name}",
-        'You recently submitted an application to add or remove dependents from your VA benefits (VA Form 28-686c/674)'
+        "Dear #{user.first_name} #{user.last_name}",
+        "We're sorry. Something went wrong when we tried to submit your application to add or remove a dependent"
```

Comment:
This is a tiny, non-blocking suggestion:

If we make use of Rails built-in i18n, we can use the i18n variables in the ERB template and the spec, such that if we ever have to update the string again, we won't have to update the spec.

Reply:
That's awesome. Great suggestion.